Add walkthrough in SearchPreview

jyash97 opened the pull request:
Demo:
![tutorial](https://user-images.githubusercontent.com/22376783/48474600-e302a900-e820-11e8-9445-11588464366b.gif)

metagrover commented on components/SearchSandbox/index.js:
```diff
@@ -18,8 +19,75 @@ const wrapper = css`
 
+/* eslint-disable-next-line */
+injectGlobal`
+	.__floater__body>div>div{
```
Linting issues

metagrover commented on components/SearchSandbox/index.js:
```diff
@@ -18,8 +19,75 @@ const wrapper = css`
+
 export const SandboxContext = React.createContext();
 
+const joyrideSteps = [
```
We should move this to a separate file

jyash97 replied:
Sure 👍

jyash97 commented:
**New Changes**
- Change position of button to near `help` button
- added `tooltip` and `message` to show status to user
- Make Joyride component reusable - `Walkthrough` 
- Introduced localStorage to track is user has already completed tutorial
- `showWalkthrough` prop is used to show button, `case` Mappings we only show for `paid` user

**Demo**
![new walkthrough](https://user-images.githubusercontent.com/22376783/48535925-f0786b80-e8d2-11e8-94d1-3ba83f33b3b3.gif)

@siddharthlatest @metagrover
